gdal/swig/python/scripts/gdal_calc.py

```diff
@@ -641,9 +647,8 @@ def main():
     parser.add_option("--optfile", dest="optfile", metavar="optfile", help="Read the named file and substitute the contents into the command line options list.")
 
     # parser.add_option("--color_table", dest="color_table", help="color table file name")
-    # when extent don't agree: 0=ignore(check only dims)/1=fail (gt must also agree)/2=union/3=intersection/GeoRectangle
-    # parser.add_option("--extent", dest="extent", type=int,
-    #                   help="how to treat different geotrasnforms [0=ignore|1=fail|2=union|3=intersect]")
+    parser.add_option("--extent", dest="extent", type=int,
```

Comment:
can we use a string value (--extent=ignore / --extent=fail) and map to the constants afterwards ? The numeric values in the user interface are not very user friendly